front/src/views/Boxes/components/Checkbox.tsx

```diff
@@ -26,6 +24,7 @@ const useCombinedRefs = (...refs): React.MutableRefObject<any> => {
   return targetRef;
 };
 
+// eslint-disable-next-line react/display-name
```

Comment:
why is it complaining here?

Reply:
It's complaining about missing the component display name, I imagine some conflict with `forwardRef`.